sdk/python/kfp/components/types/type_annotations_test.py

```diff
@@ -31,16 +31,30 @@
 class AnnotationsTest(parameterized.TestCase):
 
     def test_is_artifact_annotation(self):
-        self.assertTrue(type_annotations.is_artifact_annotation(Input[Model]))
-        self.assertTrue(type_annotations.is_artifact_annotation(Output[Model]))
         self.assertTrue(
-            type_annotations.is_artifact_annotation(Output['MyArtifact']))
-
-        self.assertFalse(type_annotations.is_artifact_annotation(Model))
-        self.assertFalse(type_annotations.is_artifact_annotation(int))
-        self.assertFalse(type_annotations.is_artifact_annotation('Dataset'))
-        self.assertFalse(type_annotations.is_artifact_annotation(List[str]))
-        self.assertFalse(type_annotations.is_artifact_annotation(Optional[str]))
+            type_annotations.is_Input_Output_artifact_annotation(Input[Model]))
+        self.assertTrue(
+            type_annotations.is_Input_Output_artifact_annotation(
+                Input[List[Model]]))
+        self.assertTrue(
+            type_annotations.is_Input_Output_artifact_annotation(Output[Model]))
+        self.assertTrue(
+            type_annotations.is_Input_Output_artifact_annotation(
+                Output[List[Model]]))
+        self.assertTrue(
+            type_annotations.is_Input_Output_artifact_annotation(
+                Output['MyArtifact']))
+
+        self.assertFalse(
+            type_annotations.is_Input_Output_artifact_annotation(Model))
+        self.assertFalse(
+            type_annotations.is_Input_Output_artifact_annotation(int))
+        self.assertFalse(
+            type_annotations.is_Input_Output_artifact_annotation('Dataset'))
+        self.assertFalse(
+            type_annotations.is_Input_Output_artifact_annotation(List[str]))
+        self.assertFalse(
+            type_annotations.is_Input_Output_artifact_annotation(Optional[str]))
```

Comment:
nit: not your code, but the test can be converted into using parameterized test. Feel free to ignore.